com.unity.shadergraph/Editor/Importers/Importers.asmdef

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Unity.ShaderGraph.Importers",
+    "rootNamespace": "",
+    "references": [
+        "GUID:f00137e8794763a4fbbd0bcfb9bc600d",
+        "GUID:97bddd7dc8a1fe343a2f93cba0c29747",
+        "GUID:20e1af7e65b430244924993fb3219d29",
+        "GUID:18b4fb767b8f81047b7c24fdec94709e",
+        "GUID:6077d53cee304074ab132af0ea2850ae",
+        "GUID:df380645f10b7bc4b97d4f5eb6303d95"
```

Comment:
These scare me-- does a source file without a .meta still have a deterministic guid?

eg. If you copypasta'd this into a new dev environment would all these references still work?

Reply:
File no longer exists, resolving